String/ReverseWords.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+function reverse_words(string $text)
+{
+    $text         = trim($text);
+    $words        = explode(' ', $text);
+    $reverseWords = array_reverse($words);
```

Comment:
Can we do it with logic as done in reverse_words example?

Reply:
I have changed the code @AnupKumarPanwar